toolchain/parse/node_kind.def

```diff
@@ -19,7 +19,7 @@
 //     Defines a parse node for a postfix operator, with the Name as token.
 //   - CARBON_PARSE_NODE_KIND_TOKEN_LITERAL(Name, LexTokenKind)
 //     Defines a parse node that corresponds to a token that is a single-token
-//     literal. The token is wrapped for LexTokenKinds.
+//     literal. The token is wrapped for LexTokenKind.
```

Comment:
Pre-existing, but I don't understand what this sentence means, either before or after your change. Looking through the history, I think it might be referring to the token kind being wrapped in the old `CARBON_TOKEN` macro? In which case I think we can just delete this sentence, since it's no longer relevant or true.

Reply:
Removing the sentence.